pages/public_cloud/containers_orchestration/managed-rancher-service/getting-started/guide.en-gb.md

```diff
@@ -0,0 +1,276 @@
+
+4. Move to **Cluster Configuration**
+![MKS Driver Cluster Configuration](images/mksdriver_cluster_config.png)
+Provide the following parameters:
+  - the **Region** where your cluster will be deployed,
+  - the **Kubernetes Version**, only versions supported by Rancher current version are listed (you can refer to the [Official Support Matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions))
+  - the **Update Policy** you want to define for your managed cluster (for more information refer to the [Managed Kubernetes Update Policies](https://help.ovhcloud.com/csm/en-public-cloud-kubernetes-change-security-update?id=kb_article_view&sysparm_article=KB0049649) guide)
+
+
+5. Move to  **Network Configuration**
+![MKS Driver Cluster Configuration](images/mksdriver_network_configuration.png)
+
+  Provide the following parameters:
+  - the **Private Netword ID** for your MKS cluster. Use 'none' to create a cluster with nodes using Public IP or select an existing OVHcloud Public Cloud private network.
+  - the **Default vRack Gateway** (optional), leave it empty to use the default gateway of your Private Network.
+  - the **Private Network Routing As Default** state, activate this feature if you want to use an OVHcloud Managed Gateway or a custom Gateway as a single exit point for your MKS nodes.
+
+
+6. Move to **NodePools Configuration**
+![MKS Driver Cluster Configuration](images/mksdriver_nodepool_configuration.png)
+  Provide the following parameters:
+  - the **Name** of the NodePool
+  - the OVHcloud Instance **Flavor** used by this NodePool
+  - the **Autoscaling** state
+  - the **Size** of your NodePool (number of nodes that will be created)  
+  - the **Monthly Billing** state (hourly by default)
+
+  Then click on 'Add Node Pool'. You can add multiple NodePools and then manage your list of Nodepools (note that the 'delete' action button of your first NodePool is grayed out till a second one is created).
+
+7. Click on **Finish & Create Cluster**
+
+8. From the Homepage your cluster is now in 'Provisioning' state
+![MKS Driver Cluster Configuration](images/mksdriver_provisioning.png)
+From the Cluser Management page, wait for your cluster to become 'Active'
+![MKS Driver Cluster Configuration](images/mksdriver_cluster_provisioning.png)
+![MKS Driver Cluster Configuration](images/mksdriver_cluster_provisioning_available.png)
+
+  Your cluster is now fully functional. You can click on 'Explore' to manage your MKS Cluster.
+
+
+##### Use OVHcloud as an **Infrastructure Provider**
+
+
+On this part we will detail how to use Rancher to create and manage Kubernetes clusters based on [OVHcloud Public Cloud Compute Instances](https://www.ovhcloud.com/en/public-cloud/compute/).
+
+  > [!warning]
+  >
+  > Deploying to OVHcloud will incur charges. For more information, refer to the [MKS](https://www.ovhcloud.com/en/public-cloud/prices/#568) and [Compute](https://www.ovhcloud.com/en/public-cloud/prices/) pricing pages.
```

Comment:
```suggestion
  > Deploying to OVHcloud will incur charges. For more information, refer to the [Compute](https://www.ovhcloud.com/en/public-cloud/prices/) pricing pages.
```

Reply:
Why ? I though that a redirection to MKS prices (where the table is showing that masters are free) would be a good idea.

Reply:
We are in the Public Cloud driver section if I'm right so nothing to do with MKS